content/blogs/2023-08-22-generating-fake-data.md

```diff
@@ -0,0 +1,421 @@
+---
+title: "Generating Fake Data to Create End-to End Orchestration Projects"
+description: " generates dynamic fake data for end-to-end orchestration projects with Kestra, for realistic data use cases and hands-on experience"
+date: 2023-08-22T16:00:00
+category: Solutions
+author:
+  name: Benoit Pimpaud
+  image: "Bpimpaud"
+image: /blogs/2023-08-22-generating-fake-data.jpg
+---
+Welcome to the world of [Shiny Rocks](https://github.com/kestra-io/shiny_rocks)! Although this company might not exist outside Kestra it has its own purpose.
+
+We often face the challenge of presenting concepts without real data, due to privacy concerns or practicalities.
+
+Shiny Rocks is a fictional company creating smartphones. "Those shiny rocks in our pockets". It provides a means to showcase various data use cases in a realistic manner. In this post, we are not just introducing this mock company, but also offering datasets it "produces" for your hands-on experience. Plus, in the future, anticipate more of Shiny Rocks; it's going to be our gimmick to bring plausible data scenarios, especially when we dive deep into use cases with Kestra.
+
+![architecture](/blogs/2023-08-22-generating-fake-data/architecture.png)
+
+## Generating fake data ##
+
+While we admire dbt [jaffle-shop's portrayal](https://github.com/dbt-labs/jaffle_shop) of a fictional e-commerce store, we wanted something more dynamic, especially when showcasing orchestration patterns. Thus, Shiny Rocks was born. Unlike static files that don't change, Shiny Rocks comes with custom scripts that generate fresh data every day. These scripts rely on a mix of mathematical functions to simulate various trends for our imaginary company.
```

Comment:
```suggestion
Unlike static files that don't change, Shiny Rocks comes with custom scripts that generate fresh data every day. These scripts rely on a mix of mathematical functions to simulate various trends for our imaginary company.
```

cut it, there is no need to mention jaffle shop here, not relevant in this context

Reply:
alright 👍